package/lib/src/utils.dart

```diff
@@ -24,7 +25,7 @@ abstract class Utils {
           state: routeState,
         );
         if (!beamLocation.isCurrent) {
-          beamLocation.create(routeInformation);
+          beamLocation.create(routeInformation, beamParameters);
         } else {
           beamLocation.update(null, routeInformation, null, false);
```

Comment:
The third parameter here (currently `null`) is also `BeamParameters` so we can add it now properly.